flake.nix

```diff
@@ -87,6 +85,7 @@
 
             wasm32 = pkgs.mkShell.override
               {
+                # TODO: This derivations forces wasi-sdk 12 so the stdenv will have the wrong tools
                 stdenv = packages.wasm32.stdenv;
```

Comment:
@kobyhallx Should we fix this now or later?

Reply:
As we need to invest more in properly defined development shells, it's ok to address this later.